asreview/state/sqlstate.py

```diff
@@ -927,21 +851,56 @@ def get_dataset(self, columns=None):
         """
         if type(columns) == str:
             columns = [columns]
+
+        # Make sure the right columns are available to drop priors/pending.
```

Comment:
Why would those columns not be available?

Reply:
Ah, I start to understand (see also comment below). I would prefer to do the slicing with SQL. That resolves most of the challenges here.